pkg/controller/account/account_controller.go

```diff
@@ -73,22 +72,52 @@ const (
 
 var awsAccountID string
 var desiredInstanceType = "m5.xlarge"
-var coveredRegions = []string{
-	"us-east-1",
-	"us-east-2",
-	"us-west-1",
-	"us-west-2",
-	"ca-central-1",
-	"eu-central-1",
-	"eu-west-1",
-	"eu-west-2",
-	"eu-west-3",
-	"ap-northeast-1",
-	"ap-northeast-2",
-	"ap-south-1",
-	"ap-southeast-1",
-	"ap-southeast-2",
-	"sa-east-1",
+var coveredRegions = map[string]map[string]string{
+	"us-east-1": {
+		"initializationAMI": "ami-000db10762d0c4c05",
```

Comment:
These would be better suited for a config map. We have other hard coded values in the operator that could go in a configmap too, so if you want to commit this now and wait for a future refactor, I think that would be fine.

Reply:
Agree, I'll create a card for this and we can refactor later.

Reply:
:+1: